### ISSUE TITLE
Pascal datasets

### ISSUE THREAD

**Issue by Ehteshamciitwah** [juhongm999/hsnet#25]
Hello,

During the training of your model. I realized the model is using nearly 11000 images for training on fold0. While the other comparable models like PFENet are using different lists containing nearly 4000 images for fold 0. Why there is a difference between training images number. am I missing something? 
Furthermore when I train your model using PFENet pipelines (data loader). The results are very low. Need your comments.  Thank you

**Comment by juhongm999** [comment]
Where did you get the number 11000?
There are only 2286 training images (https://github.com/juhongm999/hsnet/blob/main/data/splits/pascal/trn/fold0.txt).

**Comment by Ehteshamciitwah** [comment]
Hello, Thank you for your reply. 
During the training of Fold0. we train the model on 15 classes (6-20 classes) and use 1-5 classes as validation. As Fold0.txt contain images of class(1-5) which we cannot use for training. we need to use all the images in the fold1,fold2, and fold3 txt files. Because Classes (6-20) are saved in fold1,fold2,fold3 txt file. Long story short, for Fold0 training we need to train the model on Fold1, Fold2, Fold3 txt files and validate on validation fold0 txt. So the total images in fold1, Fold2, and fold3are nearly 11000. So that's why I said why we are training on 11000 images. If i am understanding anything wrong please let me know.

**Comment by juhongm999** [comment]
Oh I'm sorry for the confusion.
I haven't looked at the PFENet dataloader code and our dataloader code bases on PMM.
We use the same trainig image list as seen here (https://github.com/Yang-Bob/PMMs/tree/master/data_list/train).

**Comment by Ehteshamciitwah** [comment]
Thank you for your quick response. I found that PFENet/OSLSM models are using a training set after excluding validation images and it contains a different list than PANEt and HSNet
But during the comparison of MIoU, is it fair to compare if models are trained on different numbers of images?

**Comment by juhongm999** [comment]
I checked the number of training images for fold 0, 
https://github.com/juhongm999/hsnet/blob/main/data/splits/pascal/trn/fold1.txt
https://github.com/juhongm999/hsnet/blob/main/data/splits/pascal/trn/fold2.txt
https://github.com/juhongm999/hsnet/blob/main/data/splits/pascal/trn/fold3.txt

The total number of images only amounts to ~1000 only.

There are duplicate images across fold 1~3 and within the same fold (as each line indicates class id to segment).

Can you doublecheck the number of training images by removing duplicates across/within fold 1~3?

Also, how does the number of training images change in PFENet if you remove the duplicates?

**Comment by Ehteshamciitwah** [comment]
Thank you for your cooperation. But I am still curious, what do you mean by 1000 images only for fold0. What about the rest of 10000 images used in training of fold0. are they all duplicates? Duplicates mean images only or labels are also the same. As one image may have multi-labels according to class. However, I will explore and let you know after finding a conclusion. Thank you
